app/stac_api/views.py

```diff
@@ -74,13 +75,15 @@ def checker(request):
     return JsonResponse(data)
 
 
-class CollectionList(generics.ListAPIView):
+class CollectionList(generics.ListAPIView, mixins.CreateModelMixin):
```

Comment:
Here I would use the `GenericAPIView` instead of `ListAPIView` as you are overwritting the `get()` method, the `ListAPIView` doesn't bring any plus value.